motoko/passwords/src/passwords_assets/public/index.js

```diff
@@ -0,0 +1,35 @@
+import passwords from "ic:canisters/passwords";
```

Comment:
Could be nice if this would do `passwords.signup` as well, esp to test what happens if I try to signup as the same username/password as someone else.